backend/tardigrade/fs.go

```diff
@@ -219,8 +219,15 @@ func NewFs(name, root string, m configmap.Mapper) (_ fs.Fs, err error) {
 	f.access = access
 
 	f.features = (&fs.Features{
-		BucketBased:       true,
-		BucketBasedRootOK: true,
+		BucketBased:             true,
+		BucketBasedRootOK:       true,
+		CanHaveEmptyDirectories: false,
+		Copy: func(ctx context.Context, src fs.Object, remote string) (fs.Object, error) {
```

Comment:
I'm surprised we need to set this manually. Maybe there are others we need to set as well?

Reply:
thb I shouldn't push this as a part of fix because its not helping. Its more that I think we should implement it to give rclone info we are not supporting this currently. I will remove this part from fix